website/static/js/filepage/index.js

```diff
@@ -63,6 +65,16 @@ var FileViewPage = {
             return false;
         });
 
+        $(document).on('fileviewpage:externalView', function() {
```

Comment:
Why is this custom event necessary? Why not just pass this callback as the `onclick` handler?